runtime/src/stakes.rs

```diff
@@ -507,9 +501,9 @@ impl Stakes<StakeAccount> {
         &self.stake_delegations
     }
 
-    pub(crate) fn highest_staked_node(&self) -> Option<Pubkey> {
```

Comment:
I have to change this function, because node_pubkey() no longer return Option. So while I have to change it, I thought I'd make it return a reference. Do you want me to make it return a copy?

Reply:
Ah true, never mind, reference is fine!